backends/intel_gpu/cmake/paddle.cmake

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+find_package(Python ${PYTHON_VERSION} REQUIRED COMPONENTS Interpreter Development)
+
+if(DEFINED ENV{PADDLE_CUSTOM_PATH})
+    set(PADDLE_DIR $ENV{PADDLE_CUSTOM_PATH})
+else()
+    set(PADDLE_DIR ${Python_SITEARCH}/paddle)
+endif()
```

Comment:
please update based on backends/npu/cmake/paddle.cmake from line 17 to 21

Reply:
done